IQSS/dataverse-frontend
# Fix/429 dataset files scroll selection

**What this PR does / why we need it**:
Fix file selection on files table with infinite scrolling from a dataset 

**Which issue(s) this PR closes**:

Closes #429 

**Special notes for your reviewer**:
N/A

**Suggestions on how to test this**:
See unexpected behavior on #429  bug description.

1. Create a Dataset with +40 files.
2. Navigate to that Dataset in the SPA.
3. Select first 10 and then click Select All button, then scroll to the bottom several times to load the rest of the files.
4. File Selection should persist.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:
N/A

**Is there a release notes update needed for this change?**:
N/A

**Additional documentation**:
N/A
